pkg/remote/aws/init.go

```diff
@@ -46,7 +46,7 @@ func Init(alerter *alerter.Alerter) error {
 	resource.AddSupplier(NewLambdaFunctionSupplier(provider.Runner().SubRunner(), lambda.New(provider.session)))
 	resource.AddSupplier(NewDBSubnetGroupSupplier(provider.Runner().SubRunner(), rds.New(provider.session)))
 	resource.AddSupplier(NewDBInstanceSupplier(provider.Runner().SubRunner(), rds.New(provider.session)))
-	resource.AddSupplier(NewVPCSecurityGroupSupplier(provider.Runner().SubRunner(), ec2.New(provider.session)))
+	resource.AddSupplier(NewVPCSecurityGroupSupplier(provider.Runner(), ec2.New(provider.session)))
```

Comment:
We need to use a new sub runner to parallelize processing like other suppliers

Reply:
I did it this way because for vpc and subnet we did it that way.

Reply:
You are right my bad we create sub runner in constructor ...